src/fparser/two/utils.py

```diff
@@ -114,6 +114,11 @@
 # when reading/writing data using unformatted IO.
 _EXTENSIONS += ["open-convert"]
 
+# While non-standard, many compilers support negative numbers, and string
+# operations in stop statements, e.g. `stop -1` or `stop str1//str2`.
+# With this extension, these statements will be allowed.
+_EXTENSIONS += ["extended-stop-format"]
```

Comment:
Perhaps s/-format/-args/ here (and elsewhere) as "format" makes me think of Fortran FORMAT statements?

Reply:
Yes, that sounds better.